V1.1.4

RoyEJohnson opened the pull request:
A few small enhancements

philschatz commented:
What kinds of things did this change? Is there a Trello card for this? I'm just trying to find out what the changes impact and from the diff it's not clear what I should be looking for.

RoyEJohnson commented:
@philschatz The changes are driven primarily by what is needed for (or broken in) os-webview:

The title-font and body-font macros are refactorings of the code that is used for the standard font-setting routines. They allow custom sizing of fonts for those rare occasions when we need something outside the prescribed set.

I added true black as an available UI color. That was needed for box shadows and overlays.

I changed form padding to zero on phones because the standard form padding is pretty large and not suitable for phones.

I added a standard box-shadow for cards (which are part of the new details page design). I fixed the tab group design per instructions from Emily.

And of course I updated the version number.

philschatz commented on core/pattern-library/colors.scss:
```diff
@@ -44,7 +44,8 @@ $ui-colors: (
     success-fg: #0064a0,
-    white: #fff
+    white: #fff,
+    black: #000
```
I don't see any overlays in the diff being changed to use the `black` color. Are other repos using the colors directly? If so, should they be? 

I am a little concerned about using the colors directly because it seems that whatever they are used for styling might be a good candidate to include in this library.

RoyEJohnson replied:
Elements in the library use black now. Point taken about candidates for inclusion in the library, but I think that a utilitarian color like black is also a likely candidate for a one-off styling decision that doesn't merit inclusion in the library. We're trying to keep such things to a minimum, but they do happen.

philschatz commented on core/pattern-library/molecules.scss:
```diff
@@ -3,6 +3,7 @@
 @import 'layout';
 
 $form-border-width: 0.1rem;
+$card-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.25);
```
Is this variable used anywhere? If not, should there be an element in this repo that uses it? It seems like something that uses a card shadow would be a good candidate to style in this repo (or this repo could provide a macro that uses the card shadow).

philschatz commented on core/pattern-library/typography.scss:
```diff
@@ -9,18 +9,29 @@ $font-specs: (
     helper-label: 1.4rem 2rem
 );
 
+@mixin title-font($size) {
```
Could elements in this repository use this mixin? It would be nice to have examples that use this code (for testing).

RoyEJohnson replied:
The set-font macro uses these macros. I don't want to style anything in the standard library a custom size.

philschatz commented:
Thanks! Could there be an entry in the style guide for the `card()` mixin? That way people can see what the card shadow would look like?
